File: Java/OfficeApp/src/office/utility/Date.java
package office.utility;

public class Date {
    private int day;
    private int month;
    private int year;

    public Date(){
        day = month = 1;
        year =2020;
    }

    public Date(int day, int month, int year){
        this.day = day;
        this.month = month;
        this.year = year;
    }

    public void display(){
        System.out.println("Date of birth : "+day+"/"+month+"/"+year);
    }
}

File: CPP/Class and Object/Date/date.cpp
#include "date.h"

Date::Date() {
	day = month = 1;
	year = 2014;
}

Date::Date(int day) {
	this->day = day;
	month = day;
	year = 2024;
}

void Date::display() {
	cout << "\n\n\t" << day << "-" << month << "-" << year;
}